client/rest-high-level/src/main/java/org/elasticsearch/client/dataframe/transforms/pivot/DateHistogramGroupSource.java

```diff
@@ -153,14 +279,13 @@ public boolean equals(Object other) {
 
         return Objects.equals(this.field, that.field) &&
```

Comment:
Please either drop "this." in this line or use it consistently in other lines as well.